yt_dlp/__init__.py

```diff
@@ -320,12 +321,25 @@ def validate_outtmpl(tmpl, msg):
         del opts.outtmpl['default']
 
     def parse_chapters(name, value):
+        def parse_timestamp(x):
+            # FIXME: Maybe there's a better way to remove parenthesis
+            x = x.replace('(', '').replace(')', '')
+
+            # FIXME: This should be smarter, e.g. 'inf-1day'?
+            if x in ('inf', 'infinite'):
+                return float('inf')
+
+            if re.match(r'[\d:]+', x):
+                return parse_duration(x)
+
+            return datetime_from_str(x, precision='second', use_utc=False).timestamp()
```

Comment:
why `use_utc=False`? All times processed by yt-dlp should be assumed to be in UTC

Reply:
Thanks for clarifying that, I thought that yt-dlp used both

It seems like a bug in unified_timestamp, I have this unit test to reproduce it 
```
self.assertEqual(unified_timestamp('2023-03-11T06:48:34.008'), 1678538914)
```

Reply:
I've added a test and comment about the "bug" and also removed the use_utc param